lib/recurly/dunning_campaign.rb

```diff
@@ -0,0 +1,33 @@
+module Recurly
+  # Recurly Documentation: https://developers.recurly.com/api-v2/latest/index.html#tag/dunning-campaigns
+  class DunningCampaign < Resource
+    # @return [[DunningCycle], []]
+    has_many :dunning_cycles
+
+    # @return [[Plan], []]
+    has_many :plans
+
+    define_attribute_methods %w(
+      id
+      name
+      code
+      description
+      default
+      dunning_cycles
+      created_at
+      updated_at
+      deleted_at
+      plans
+    )
+
+    def bulk_update(plan_codes)
```

Comment:
do we also need to support `plan_ids` here?

Reply:
For API V2 the `plan_code` is the attribute for looking up a `Plan` resource. The `id` field is not exposed for plans in API V2.